TombEngine/Renderer/ConstantBuffers/InstancedSpriteBuffer.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <SimpleMath.h>
+
+namespace TEN::Renderer
+{
+	using DirectX::SimpleMath::Matrix;
+	using DirectX::SimpleMath::Vector4;
+
+	struct alignas(4) InstancedSprite
```

Comment:
use alignas(16) to be 100% sure it is correctly aligned.
Might break in the future if the compiler decides to not align it properly